src/main/webapp/WEB-INF/view/profile.jsp

```diff
@@ -0,0 +1,85 @@
+<%--
+  Copyright 2017 Google Inc.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+--%>
+<%@ page import="java.util.List" %>
+<%@ page import="codeu.model.data.User" %>
+<%@ page import="codeu.model.data.Message" %>
+<%@ page import="codeu.model.store.basic.UserStore" %>
+
+<% User user = (User) request.getAttribute("view_user"); %>
+<!DOCTYPE html>
+<html>
+<head>
+    <title>
+        <% if(request.getAttribute("error") != null) { %>
+        User Not Found.
+        <% } else { %>
+        <%= user.getName() %>'s Profile Page
+        <% } %>
+    </title>
+    <link rel="stylesheet" href="/css/main.css" type="text/css">
+
+    <style>
+        #messages {
+            background-color: white;
+            height: 500px;
+            overflow-y: scroll
+        }
+    </style>
+
+    <script>
+        // scroll the chat div to the bottom
+        function scrollChat() {
+            var chatDiv = document.getElementById('messages');
+            chatDiv.scrollTop = chatDiv.scrollHeight;
+        };
+    </script>
+</head>
+<body onload="scrollChat()">
+
+<nav>
+    <a id="navTitle" href="/">CodeU Chat App</a>
+    <a href="/conversations">Conversations</a>
+    <% if (request.getSession().getAttribute("user") != null) { %>
+    <a>Hello <%= request.getSession().getAttribute("user") %>!</a>
+    <% } else { %>
+    <a href="/login">Login</a>
+    <a href="/register">Register</a>
+    <% } %>
+    <a href="/about.jsp">About</a>
+</nav>
+
+<div id="container">
+    <% if(request.getAttribute("error") != null){ %>
+    <h2 style="color:red"><%= request.getAttribute("error") %></h2>
```

Comment:
It's better to style this with stylesheet, so that it would be consistent. Say "<h2 class="error">...</h2>" here and add
the styling you want to the stylesheet file.